### PR TITLE
fix(infiniteGrid): fix return value of getBottomElement

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#467

## Details
- if the position of element is same, getBottomElement returns right element.
- add comment

## Preferred reviewers
@netil

### CODE REVIEW

File: src/infiniteGrid.js [hunk 1]
```diff
@@ -425,15 +425,15 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document], function($, ns, glob
 		 * @ko 레이아웃의 맨 아래에 있는 카드 엘리먼트를 반환한다.
 		 * @method eg.InfiniteGrid#getBottomElement
 		 *
-		 * @return {HTMLElement} Card element at the bottom of a layout <ko>레이아웃의 맨 아래에 있는 카드 엘리먼트</ko>
+		 * @return {HTMLElement} Card element at the bottom of a layout (f the position of card element is same, returns right element)<ko>레이아웃의 맨 아래에 있는 카드 엘리먼트 (카드의 위치가 같은 경우, 오른쪽 엘리먼트가 반환된다)</ko>
```
Comment (netil) [inline src/infiniteGrid.js]: `if the position of card elements are same, it returns the last right element`

File: src/infiniteGrid.js [hunk 2]
```diff
@@ -395,7 +395,7 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document], function($, ns, glob
 		 * @ko 레이아웃의 맨 위에 있는 카드 엘리먼트를 반환한다.
 		 * @method eg.InfiniteGrid#getTopElement
 		 *
-		 * @return {HTMLElement} Card element at the top of a layout <ko>레이아웃의 맨 위에 있는 카드 엘리먼트</ko>
+		 * @return {HTMLElement} Card element at the top of a layout. (if the position of card element is same, returns left element) <ko>레이아웃의 맨 위에 있는 카드 엘리먼트 (카드의 위치가 같은 경우, 왼쪽 엘리먼트가 반환된다)</ko>
```
Comment (netil) [inline src/infiniteGrid.js]: `if the position of card elements are same, it returns the first left element`

Comment (sculove) [inline src/infiniteGrid.js]: I applied it

File: src/infiniteGrid.js (hunk 1 above)
Comment (sculove) [inline src/infiniteGrid.js]: I applied it

**Pull request merged**